geocity/wsgi.py

```diff
@@ -10,9 +10,7 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
-from whitenoise import WhiteNoise
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "geocity.settings")
 
 application = get_wsgi_application()
-application = WhiteNoise(application, root=os.environ["STATIC_FILES_ABSOLUTE_PATH"])
```

Comment:
how are static files served on production then ?

Reply:
It seems it's done by `whitenoise.middleware.WhiteNoiseMiddleware` (see http://whitenoise.evans.io/en/stable/#quickstart-for-django-apps)

Seems that wrapping the wsgi app was not necessary in the first place (it's for non-django apps)

Locally at least it still works with that change.